core/trino-main/src/main/java/io/trino/execution/buffer/PageCodecMarker.java

```diff
@@ -20,7 +20,7 @@
 import static com.google.common.base.Preconditions.checkArgument;
 
 /**
- * Encodes boolean properties for {@link SerializedPage} by using a bitmasking strategy, allowing
+ * Encodes boolean properties for serialized page by using a bitmasking strategy, allowing
```

Comment:
nit: `PageCodecMarker` could be made private enum within PagesSerde now (modulo the `TestPageCodecMarker`). Still you can make it package private now.